manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala

```diff
@@ -225,34 +224,34 @@ Note that you can not introduce new variables here. Although it might look very
 first will produce a subgraph for every path it can find between `a` and `b`, and the latter will eliminate any matched
 subgraphs where `a` and `b` do not have a directed relationship chain between them.
              """,
-      queryText = """match (tobias {name: 'Tobias'}), (others) where others.name IN ['Andres', 'Peter'] and (tobias)<--(others) return others""",
-      optionalResultExplanation = "Nodes that have an outgoing relationship to the \"+Tobias+\" node are returned.",
+      queryText = """MATCH (tobias {name: 'Tobias'}), (others) WHERE others.name IN ['Andres', 'Peter'] AND (tobias)<--(others) RETURN others""",
+      optionalResultExplanation = "Nodes that have an outgoing relationship to the \"Tobias\" node are returned.",
       assertions = (p) => assertEquals(List(Map("others" -> node("Andres"))), p.toList))
   }
 
   @Test def predicate_with_properties() {
     testQuery(
       title = "Filter on patterns with properties",
       text = """You can also add properties to your patterns:""",
-      queryText = """match (n) where (n)-[:KNOWS]-({name:'Tobias'}) return n""",
-      optionalResultExplanation = """Finds all nodes that have a +KNOWS+ relationship to a node with the name "+Tobias+".""",
+      queryText = """MATCH (n) WHERE (n)-[:KNOWS]-({name: 'Tobias'}) RETURN n""",
+      optionalResultExplanation = """Finds all nodes that have a `KNOWS` relationship to a node with the name "Tobias".""",
       assertions = (p) => assertEquals(List(Map("n" -> node("Andres"))), p.toList))
   }
 
   @Test def has_not_relationship_to() {
     testQuery(
       title = "Filter on patterns using NOT",
       text = """The `NOT` function can be used to exclude a pattern. """,
```

Comment:
Is `NOT` a function? If it is, then I suppose the pattern expression `(persons)-->(peter)` is a function argument in this query. Should `NOT` be written with parentheses and in lower camel case like other functions: `not((persons)-->(peter))`? Sorry, this is out of scope for your PR, but I'm curious.